docs/src/examples/components/Button/Variations/index.tsx

```diff
@@ -24,6 +24,11 @@ const Variations = () => (
       description="A button can be shown in form of a text to indicate some less-pronounced actions."
       examplePath="components/Button/Variations/ButtonExampleText"
     />
+    <ComponentExample
+      title="Size"
+      description="A button can have assorted sizes."
```

Comment:
It would be nice to consolidate the description of the size in the components' examples. We have different for Avatar, Text, Button.. :\